Documentation/data-collection.md

```diff
@@ -703,11 +703,16 @@ data:
     # and per volume type (filesystem/block)
     - '{__name__="cluster:kube_persistentvolume_plugin_type_counts:sum"}'
     #
-    # owners: (https://github.com/open-cluster-management, @open-cluster-management/cluster-lifecycle-admin)
+    # owners: (https://github.com/orgs/stolostron/teams/server-foundation, @acm-server-foundation)
     #
     # acm_managed_cluster_info provides Subscription watch and other information for the managed clusters for an ACM Hub cluster.
     - '{__name__="acm_managed_cluster_info"}'
     #
+    # owners: (https://github.com/orgs/stolostron/teams/server-foundation, @acm-server-foundation)
+    #
+    # acm_managed_cluster_worker_cores:sum tracks the number of CPU cores on the worker nodes of the ACM managed clusters.
+    - '{__name__="acm_managed_cluster_worker_cores:sum"}'
```

Comment:
1. what's the prometheus expr for recording rule `acm_managed_cluster_worker_cores:sum`, how the metric is calculated
2. https://github.com/rhobs/configuration/blob/main/configuration/telemeter/metrics.json did not see `acm_managed_cluster_worker_cores:sum` added

Reply:
`acm_managed_cluster_worker_cores:sum` is a recording rule belongs to ACM. That will be created once ACM 2.11+ is installed on an OCP cluster. Here is the definition of this recording rule: https://github.com/stolostron/backplane-operator/pull/718.

Should I add `acm_managed_cluster_worker_cores:sum` into https://github.com/rhobs/configuration/blob/main/configuration/telemeter/metrics.json?

Reply:
> Should I add `acm_managed_cluster_worker_cores:sum` into https://github.com/rhobs/configuration/blob/main/configuration/telemeter/metrics.json?

yes, should add the metric to rhobs repo, otherwise, you can not find the metrics from telemeter server even you can find it in your local cluster, see `Synchronize the Telemeter server’s configuration` part of https://rhobs-handbook.netlify.app/products/openshiftmonitoring/telemetry.md/

Reply:
Why using sum() and not max()?
IIUC the source metric comes from https://github.com/stolostron/clusterlifecycle-state-metrics/ and though sum and max should return the same result, max() is a better aggregation method, just in case Prometheus scrapes 2 instances by accident.

Reply:
That makes sense. I'll update the recording rule on the ACM side and then this PR.

Reply:
The update on ACM side is done and this PR has been updated as well.